src/javascripts/ng-admin/Main/component/provider/HttpErrorService.js

```diff
@@ -0,0 +1,36 @@
+const HttpErrorService = ($state, $translate, notification) => ({
+    handleError: function(event, toState, toParams, fromState, fromParams, error) {
+        switch (error.status) {
+            case 404:
+               this.handle404Error(event, error);
+               break;
+            case 403:
+               this.handle403Error(error);
```

Comment:
if you do this I get sonar style check errors because the parameters are not used in the method (Unused function parameters should be removed javascript : UnusedFunctionArgument)
Nothing is mentionned about this in google guidelines